openedx_events/learning/signals.py

```diff
@@ -401,3 +402,51 @@
         "badge": BadgeData,
     }
 )
+
+
+# .. event_type: org.openedx.learning.idv_attempt.created.v1
+# .. event_name: IDV_ATTEMPT_CREATED
+# .. event_description: Emitted when a IDV attempt is created by an IDV vendor
```

Comment:
```suggestion
# .. event_description: Emitted when a IDV attempt is created
```

I'd recommend keeping it even more generic by removing "by an IDV vendor" from the event descriptions. This is because some forms of IDV are created by humans and not vendors (e.g. `ManualVerification`).